### DISCUSSION TITLE
[5 giugno 2023] Complessità

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/automi-calcolabilita-complessita#12] [Complessità]
![image](https://github.com/sapienzastudentsnetwork/acc2324/assets/69389461/713f527b-a953-4829-befe-461345172db6)

**Comment by Exyss** [comment]
Per il primo punto, ripropongo la soluzione presente sui miei appunti

![image](https://github.com/sapienzastudentsnetwork/acc2324/assets/65043207/f5370d63-1a3e-4334-a822-14fda32754ba)
![image](https://github.com/sapienzastudentsnetwork/acc2324/assets/65043207/3d906d6f-46f9-43bb-ac35-9511dff1f1e6)
![image](https://github.com/sapienzastudentsnetwork/acc2324/assets/65043207/db05b22e-503f-4356-822a-36d61d928177)

-----

La dimostrazione richiesta nel secondo punto è presente sul libro di testo

**Comment by daromacarlo** [comment]
Può essere una soluzione?

<img width="930" height="753" alt="Immagine 2026-01-28 135428" src="https://github.com/user-attachments/assets/426e965d-fca0-412e-b031-a89d9fa381f5" />
<img width="930" height="824" alt="Immagine 2026-01-28 135450" src="https://github.com/user-attachments/assets/70f6d86a-0a02-40f3-96e1-80bfa090c0f5" />
